docs/_docs/snapshots/snapshots.adoc

```diff
@@ -146,29 +146,23 @@ See the link:tools/control-script#checking-snapshot-consistency[Control Script]
 commands.
 
 == Restoring From Snapshot
-
-Currently, the data restore procedure has to be performed manually. In a nutshell, you need to stop the cluster,
-replace persistence data and other files with the data from the snapshot, and restart the nodes.
+=== Manual snapshot restore procedure
+In a nutshell, you need to stop the cluster, replace persistence data and other files with the data from the snapshot, and restart the nodes.
 
 The detailed procedure looks as follows:
 
 . Stop the cluster you intend to restore
 . Remove all files from the checkpoint `$IGNITE_HOME/work/cp` directory
-. Do the following on each node. Clean the
-link:link:persistence/native-persistence#configuring-persistent-storage-directory[`db/{node_id}`] directory separately if
-it's not located under the Ignite `work` dir:
-    - Remove the files related to the `{nodeId}` from the `$IGNITE_HOME/work/db/binary_meta` directory
-    - Remove the files related to the `{nodeId}` from the `$IGNITE_HOME/work/db/marshaller` directory
-    - Remove the files and sub-directories related to the `{nodeId}` under your `$IGNITE_HOME/work/db` directory. Clean the
-    - Copy the files belonging to a node with the `{node_id}` from the snapshot into the `$IGNITE_HOME/work/` directory.
-If the `db/{node_id}` directory is not located under the Ignite `work` dir then you need to copy data files there.
+. Do the following on each node. Clean the link:link:persistence/native-persistence#configuring-persistent-storage-directory[`db/{node_id}`] directory separately if it's not located under the Ignite `work` dir:
```

Comment:
We should not have many things to do in one item, lets split it a bit. Otherwise people will skip over after reading the first sentence.

For example:
. On each node:
    - If the link:link:persistence/native-persistence#configuring-persistent-storage-directory[`db/{node_id}`] directory is not located under the Ignite `work` dir, clean it.

Reply:
Done, thanks.